client/js/models/archive.js

```diff
@@ -3,7 +3,12 @@ module.exports = {
   namespace: 'archive',
   state: {
     key: null,
-    file: null
+    file: null,
+    numPeers: 0,
+    signalhubs: [
+      'signalhub.mafintosh.com',
+      'signalhub.dat.land'
```

Comment:
signalhubs will be fed in from the server with the rehydration state. different dats might have different signalhubs it is associated with so we'll need to find that out from the server to speed up discovery. karissa and max and i discussed this a while back.